src/components/overview/cart.js

```diff
@@ -49,26 +46,70 @@ export default function Cart() {
     setSelected(e.target.value);
   }
 
+  //Change current first SKU based on size
+  function setSkuOnFirstChange(e) {
```

Comment:
```suggestion
  function setSkuOnFirstChange(e, setHook) {
    var skus = Object.keys(stateVal.currentStyle.skus);
    setSku(parseInt(skus[e.target.value]));
    setHook(e.target.value);
  }
```

Reply:
Possibly could consolid serSkoOnFirstChange with setSkuOnChange by adding the setHook argument.